src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Figure extends FigureSupplier implements Draw {
```

Comment:
If `Figure` class extends FigureSupplier, then all its child classes also extends FigureSupplier. It is not very logical, for example, in the `Circle` class, to call the method `getRandomFigure()` that returns instance of another class(for example Square). Let `Figure` class implements interfaces Drawable and AreaCalculator. And let's made FigureSupplier class as  stand alone.
```suggestion
public class Figure implements AreaCalculator, InformationDrawer {
```